cpu/native/Makefile.dep

```diff
@@ -21,6 +21,11 @@ ifneq (,$(filter periph_rtc,$(USEMODULE)))
   USEMODULE += xtimer
 endif
 
+ifneq (,$(filter eui_provider,$(USEMODULE)))
+  USEMODULE += native_eui_provider
+  USEMODULE += l2util
```

Comment:
Isn't that rather a dependency for `native_eui_provider` (which I also would prefer to be renamed to `native_cli_eui_provider`.